pycbc/filter/qtransform.py

```diff
@@ -220,6 +224,10 @@ def qseries(fseries, Q, f0):
     cenergy = TimeSeries(zeros(tlen, dtype=numpy.complex128),
```

Comment:
I can't add a comment on line 219 (wenergy = etc.) but if I could it would ask why this has energy in the name.  It's a windowed Fourier transform ..
Could we just say `windowed = numpy.roll(windowed, -center)` ? then no need for the (to me) confusing name. 
then also suggest cenergy change to ctseries (complex timeseries) or whatever